solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -263,6 +263,26 @@ pub mod solana_ibc {
             .map_err(error::Error::ContextError)
             .map_err(|err| error!((&err)))
     }
+
+    #[allow(unused_variables)]
+    pub fn send_transfer(
+        ctx: Context<SendTransfer>,
+        port_id: ibc::PortId,
+        channel_id_on_b: ibc::ChannelId,
+        base_denom: String,
+        msg: ibc::MsgTransfer,
+    ) -> Result<()> {
+        let mut store = storage::from_ctx!(ctx, with accounts);
+        let mut token_ctx = store.clone();
+
+        ::ibc::apps::transfer::handler::send_transfer(
```

Comment:
`ibc::aps::transfer` should work (without the `::` in front. Here and throughout the PR.

Reply:
cool